### PR TITLE
Created and finished Desktop Version - Used Media Querys to aling all the contents for all the sections (See comment in PR to get the specifics)

### PR DESCRIPTION
**Pull request by hernandanielzamora**
> Changes made in the HTML file: 
  - Created some container tags (divs mostly) To provide the page with a better structure and be able to align items correctly in the desktop version and not alter the mobile version. 

> Changes made in the CSS file: 
  - Used media query  in order to create and accommodate items for the desktop version, breakpoints used (min-width -> max-width): 
      - 768px -> 1024px
      - 1024px -> 1440px
      - 1440px -> 2560px
      - 2560px -> undefined
   
  - All the CSS changes made in the breakpoints use responsive measures (%, em, vw, vh).
  - I choose this many breakpoints to give more responsiveness options.
  - Same classes as the mobile version were used.
  - The whole CSS code is properly commented on in order to be more readable.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,6 +26,12 @@
 				<div class="bar"></div>
 				<div class="bar"></div>
 				<div class="bar"></div>
+				<ul class="nav-list">
+					<li class="nav-item"><a href="">Portfolio</a></li>
+					<li class="nav-item"><a href="">About</a></li>
+					<li class="nav-item"><a href="">Contact</a></li>
+					<li class="nav-item"><a href=""><img src="media/mail-icon.png" alt="Mail Icon" class="mail-icon"></a></li>
+				</ul>
```
Comment (ouasamine) [inline index.html:34]: - [ ] Nice work implementing the navigation menu in the desktop version, however, the email icon is not currently being displayed correctly as shown below 🔽, please make sure to include the icon in your repo as it is not existent right now

### Figma Design
Figma nav
### Your Design
user design nav

File: index.html [hunk 2]
```diff
@@ -34,13 +40,15 @@
...
-			<p class="main-text">I can help you build a product , feature or website Look through some of my work and
-				experience!
-				If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+			<div class="text-container">
+				<h1 class="main-title">Hey There. I'm Hernán</h1>
+				<h2 class="main-subtitle">I'am a Software Developer</h2>
+				<p class="main-text">I can help you build a product , feature or website Look through some of my work and
+					experience!
+					If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+			</div>
```
Comment (ouasamine) [inline index.html:49]: - [ ] Well done designing the desktop structure of this section however, you missed to add a background image as shown below 🔽 

### Figma Design
Figma header

### Your Design
user header design

File: index.html [hunk 3]
```diff
@@ -59,133 +67,197 @@ <h2 class="main-subtitle">I'am a Software Developer</h2>
 
 		<!-- Works Section -->
 		<section class="work-section">
-			<h2 class="work-title">My Recent Works</h2>
-			<div class="work-bar"></div>
+			<div class="work-text-container">
```
Comment (ouasamine) [inline index.html:70]: - [ ] Please note the differences between your design and the Figma template shown below 🔽 as you need to add more spacing on the sides and reduce the space between the text and the line to match the Figma design, and you also need to pick the correct color.

### Figma Design
Figma recent w
### Your Design
you recent w

File: index.html [hunk 4]
```diff
@@ -59,133 +67,197 @@ <h2 class="main-subtitle">I'am a Software Developer</h2>
...
-							<li class="project-ruby"><button class="project-feature-btn" disabled="disabled"
-									type="button">Ruby</button></li>
-						</ul>
+			<!-- First Cards -->
+			<div class="cards-container">
+				<div class="card-container card-1">
+					<div class="card-content card-1-content">
+						<h3 class="project-title">Profesional Art Printing Data</h3>
+						<p class="project-text">A daily selection of privately personalized reads; no accounts or sign-ups required.
+							has been the industry's standard.</p>
```
Comment (ouasamine) [inline index.html:101]: - [ ] This section currently is not shown correctly, I can only see one card and it's not displayed correctly and doesn't have the required interactions (text and dark opacity should disappear upon cursor hover), please refer to the images below 🔽 

### Figma Design
Figma cards
### Your Design
your cards

File: index.html [hunk 5]
```diff
@@ -195,16 +267,16 @@ <h2 class="aboutme-title">About me</h2>
 				<p class="aboutme-text">Hello I'm a software developer! I can help you build a product , feature or website Look
 					through some of my work and experience! If you like what you see and have a project you need coded, don't
 					hestiate to contact me.</p>
```
Comment (ouasamine) [inline index.html:269]: - [ ] Please make sure that this section matches the one on the Figma design, as it is missing a background at the moment like shown below 🔽 

### Figma Design
figma about

### Your Design
your about

**Review by ouasamine — CHANGES_REQUESTED** [review]
Hi @hernandanielzamora 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Implemented design endpoints successfully ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @hernandanielzamora 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️

- You did a great job by adding the background image in your about section but kindly make sure that you reduce the a bit to the size so that it will  make your design stick more to the Figma design also adjust a bit the disposition

Your design 
image


Figma design
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by IngDavidTM — APPROVED** [review]
Hi @hernandanielzamora  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**